source/ways-of-working/troubleshooting/index.html.md.erb

```diff
@@ -111,6 +113,106 @@ weight: 99
    > Although you may have permissions to delete a helm release or pod in a non-production environment, use this privilege wisely as you could be _hiding a potential bug_ which could also _occur in production_. 
 
+###
+> Please see [V1 docs](#flux-v1) for troubleshooting an issue with resources managed through Flux V1.
+
+### Latest image is not updated in cluster
+
+- Start with checking [cnp-flux-config](https://github.com/hmcts/cnp-flux-config) to make sure flux has updated/ committed the image. 
+- If image hasn't been committed to Github, see [ Flux did not commit latest image to Github](#flux-did-not-commit-latest-image-to-github).
+- If flux has committed the new image to Github, check if the `HelmRelease` has been updated by Flux. Run below command and check that the image tag has been updated in the output
+
+    ```shell
+    kubectl get helmreleases.helm.toolkit.fluxcd.io -n <your-namespace> <your-helm-release-name> -o yaml
+    ```
+- If Image is not updated in above, [Change in git is not applied to cluster](#change-in-git-is-not-applied-to-cluster).
+- If the image tag is updated and still application pods are not deployed, see [Updated HelmRelease is not deployed to cluster](#updated-helmrelease-is-not-deployed-to-cluster)
+
+### Flux did not commit latest image to Github
+
+   - Image automation is run from management cluster (CFTPTL). Please login to cftptl cluster before further troubleshooting.
+   - Image reflector controller keeps polling ACR for new images, but it should generally update the new image in 10 minutes.
+   - Check status of `imagerepositories` and verify the last scan.
+
+    ```shell
+    kubectl get imagerepositories -n flux-system  <repository name(usually helm release name)> 
+    ```
+   - If the last scan doesn't update, check image reflector controller logs to see if there any logs related to the helm repo.
+
+    ```shell
+    kubectl logs -n flux-system -l app=image-reflector-controller
+    # search for specific image
+    kubectl logs -n flux-system -l app=image-reflector-controller | grep <Release Name>
+    ```
+   - If the last scan is latest, check `imagepolicy` status to verify that the image returned matches the expectation.
+
+    ```shell
+    kubectl get imagepolicies -n flux-system <policy name(usually helm release name)>
+    ```
+   - If it doesn't match the expected tag, verify image reflector controller logs as described above.
+   - If the `imagepolicy` object returned shows the expected image, but it didn't commit to Github, check image automation controller logs.
+
+    ```shell
+    kubectl logs -n flux-system -l app=image-automation-controller
+    # search for specific image
+    kubectl logs -n flux-system -l app=image-automation-controller | grep <Release Name>
+    ```
+
+### Updated HelmRelease is not deployed to cluster
+
+   - Helm operator queues all the updates, so it could take up to 20 minutes sometimes to be picked up.
```

Comment:
Ah this is still a problem on flux v2 is it better? 

Operator per namespace still on the cards?

Reply:
yes, operator per ns is still on cards. Its actually slightly better from what we see as helm chart responsibility is taken to source operator.